backend/apps/web_server/lib/room.ex

```diff
@@ -0,0 +1,43 @@
+defmodule WebServer.Room do
+    def start_link(room_name) do
+        "Room: #{room_name} started" |> IO.puts
+        Agent.start_link(fn -> [] end, name: room_name)
+    end
+
+    def join_to_room(room_name, pid) do
+        Agent.update(room_name, fn state ->
+            add_to_room(state, pid)
+        end)
+    end
+
+    def kick_from_room(room_name, pid) do
+        Agent.update(room_name, fn state ->
+            remove_from_room(state, pid)
+        end)
+    end
```

Comment:
Processes names are added for allowing easy finding them so collition with name can happen. In this subject i just know there will be one romm but in general it should be done in another way